explorer/src/lib/services/duskAPI.js

```diff
@@ -54,11 +54,8 @@ const toHeadersVariables = unless(
  * @param {{ query: string, variables?: Record<string, string | number> }} queryInfo
  */
 const gqlGet = (queryInfo) =>
-  fetch(makeNodeUrl("/02/Chain"), {
-    body: JSON.stringify({
-      data: queryInfo.query,
-      topic: "gql",
-    }),
+  fetch(makeNodeUrl("/on/graphql/query"), {
+    body: queryInfo.query.replace(/\s+/g, " ").trim(),
```

Comment:
Why is not possible to use `.query` directly?

Reply:
> Why is not possible to use `.query` directly?

I actually can deconstruct `queryInfo`. There's also a `variables` property on it that is being used to set headers.

Reply:
I think @herr-seppia meant that the trimming and replacing multiple spaces is not necessary (and I agree).
About the variables: does the "on" endpoint accept them the same way as before?

Reply:
> I think @herr-seppia meant that the trimming and replacing multiple spaces is not necessary (and I agree).

Ah... I did that to "minify"/normalize what is sent to the server. 

>  About the variables: does the "on" endpoint accept them the same way as before?

It does. See: https://github.com/dusk-network/rusk/pull/3421/files#diff-50cd76c681e016a6ef7cf4a25cd32b22b2c80c9e64a04a123c6da06f0d6f4b76R35